interfaces/builtin/basedeclaration_test.go

```diff
@@ -179,6 +179,7 @@ func (s *baseDeclSuite) TestAutoConnectPlugSlot(c *C) {
 	snowflakes := map[string]bool{
 		"content":     true,
 		"home":        true,
+		"lxd":         true,
```

Comment:
I'm not sure why lxd is a snowflake in TestAutoConnectPlugSlot. Can you comment?